packages/thirdweb/package.json

```diff
@@ -152,6 +152,7 @@
     "@coinbase/wallet-sdk": "3.7.2",
     "@emotion/react": "11.11.4",
     "@emotion/styled": "11.11.0",
+    "@google/model-viewer": "^2.1.1",
```

Comment:
@MananTank pretty sure there's a 3.0 of this - can we upgrade as part of the port?

also let's pin specific versions please

Reply:
done